app/components/repository-sidebar.js

```diff
@@ -116,24 +82,19 @@ export default Component.extend({
     }
   }),
 
-  tab: alias('tabStates.sidebarTab'),
+  tab: reads('tabStates.sidebarTab'),
+  isTabRunning: equal('tab', SIDEBAR_TAB_STATES.RUNNING),
+  isTabSearch: equal('tab', SIDEBAR_TAB_STATES.SEARCH),
 
-  repositoryResults: computed('tab', 'repositories.{searchResults.[],accessible.[]}', function () {
-    let tab = this.tab;
-    let searchResults = this.get('repositories.searchResults.[]');
-    let accessible = this.get('repositories.accessible.[]');
-    let results = accessible;
+  repositoryResults: computed('isTabSearch', 'repositories.{searchResults.[],accessible.[]}', function () {
```

Comment:
I'm not sure if this construction `repositories.{searchResults.[],accessible.[]}` works properly in all cases. Seems a bit unreliable to me. But maybe that's just me 😄

Reply:
Yeah, it did seem weird to me! I wasn't sure how much I wanted to refactor stuff that isn't directly related to the issue, but I already did a little, so maybe I'll take it a bit further

Reply:
Made some changes